eng/pipelines/templates/jobs/archetype-sdk-integration.yml

```diff
@@ -53,6 +55,32 @@ jobs:
     steps:
       - template: ../steps/common.yml
 
+      - ${{ if eq(parameters.ProvisionScopedResources, true) }}:
+          - pwsh: |
+              # $uniqueIdentifier generates a pseudorandom value that represents a unique name that can be used with all resources
+              $uniqueIdentifier =  "t" + [guid]::NewGuid().ToString().Replace("-", "").Substring(0,16)
```

Comment:
We should consider using a unique value that is related to the build id that we are using so we have a chance to map resource names back to the job that created them.